<?xml version="1.0" encoding="utf-8"?>
<launch>
  <!-- Node -->
  <node name="pub_human_tf" pkg="approach_person" type="pub_human_tf.py" output="screen" />
  <node name="human_coord_generator" pkg="approach_person" type="human_coord_generator.py" output="screen" />
  <node name="approach_person_server" pkg="approach_person" type="approach_person_server.py" output="screen" />

  <rosparam file="$(find happymimi_params)/location/tmp_human_location.yaml" command="load" ns="/tmp_human_location" />

</launch>
